frontend/packages/knative-plugin/src/utils/knative-topology-utils.ts

```diff
@@ -421,19 +430,20 @@ export const tranformKnNodeData = (
  */
 export const filterNonKnativeDeployments = (resources: DeploymentKind[]): DeploymentKind[] => {
   const KNATIVE_CONFIGURATION = 'serving.knative.dev/configuration';
-  const KNATIVE_EVENTS_CRONJOB = 'sources.eventing.knative.dev/cronJobSource';
-  const KNATIVE_EVENTS_CONTAINER = 'sources.eventing.knative.dev/containerSource';
-  const KNATIVE_EVENTS_APISERVER = 'sources.knative.dev/apiServerSource';
-  const KNATIVE_EVENTS_CAMEL = 'sources.eventing.knative.dev/camelSource';
-  const KNATIVE_EVENTS_KAFKA = 'sources.eventing.knative.dev/kafkaSource';
+  const allEventSourceModels: K8sKind[] = [
+    EventSourceCronJobModel,
+    EventSourceContainerModel,
+    EventSourceApiServerModel,
+    EventSourceCamelModel,
+    EventSourceKafkaModel,
+    EventSourceServiceBindingModel,
+  ];
```

Comment:
we need to see if there is a way to handle it in a generic way i.e without have check for specific models for ES as dynamic sources will in introduced soon. Then this may not work

Reply:
We still don't have an solution for dynamic event sources, so dealing with the static set is ok for now.

Reply:
i suggested to create a util which will roll up uid's for eventsource and then check for it against ownerefrences in deployments if they match filter it out, I think this should do what we want instead of using all ES models.

Reply:
I have used the `knEventSources` which was already present in the scope, passed it to the `filterNonKnativeDeployments` function as an optional argument to use as reference to filter the deployments for event sources.